plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -0,0 +1,196 @@
+package pgbouncer
+
+import (
+	"bytes"
+	"database/sql"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/lib/pq"
```

Comment:
@mced that doesn't make any sense both use the postgresql wireformat and use queries, otherwise the code in this PR wouldn't work.

I bet if you tried postgresql_extensible with the version code disabled you could do everything this plugin is doing.